doc/nrf/includes/external_flash_nrf91.txt

```diff
@@ -0,0 +1,11 @@
+To use the external flash memory, you must program the :ref:`nrf9160_ug_intro` by completing the following steps:
```

Comment:
This text usually follows the table of supported platforms. If the table contains both Thingy:91 and nRF9160 DK, it's not clear that the above text refers to nRF9160 DK only.